gammapy/utils/regions.py

```diff
@@ -148,7 +148,14 @@ def f(x, coords):
         center = SkyCoord(lon * u.deg, lat * u.deg)
         return np.sum(center.separation(coords).deg)
 
-    result = minimize(f, x0=[0, 0], args=(positions,), bounds=[(-180, 180), (-90, 90)])
+    eps = 1e-5
```

Comment:
This change seems reasonable, but unrelated? Did you see any test fails that motivate the change?

Reply:
yes it was failing on my machine

Reply:
OK, interesting...this fail didn't appear in any of our CI builds. I guess one could choose the bounds even more reasonable, as the point must lie inside the polygon spanned by the others, but I can do this in a follow up PR. So just leave as is...